core/domain/takeout_service.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions to export the data of all user related models from a given
+user_id.
+"""
+from __future__ import absolute_import   # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.platform import models
+
+(
+    user_models, collection_models, exploration_models, story_models,
+    feedback_models, suggestion_models,
+    email_models) = models.Registry.import_models([
+        models.NAMES.user, models.NAMES.collection, models.NAMES.exploration,
+        models.NAMES.story, models.NAMES.feedback, models.NAMES.suggestion,
+        models.NAMES.email])
+
+
+def export_all_models(user_id):
+    """Exports all models according to model defined export_data functions."""
+    # User models.
+    stats_data = user_models.UserStatsModel.export_data(user_id)
```

Comment:
Can we create this method such that we don't have to explicitly call `.export_data` from each model? Maybe we can create a list of `User data takeout Models` and loop over it to call export_data as well as adding data to the dict?

(This way in future we just have to add the new model class in `User data takeout Models` list.)

Reply:
done